webknossos/webknossos/utils.py

```diff
@@ -103,3 +105,22 @@ def time_since_epoch_in_ms() -> int:
     d = datetime.utcnow()
     unixtime = calendar.timegm(d.utctimetuple())
     return unixtime * 1000
+
+
+def copy_directory_with_symlinks(
+    src_path: Path,
+    dst_path: Path,
+    ignore: Iterable[str] = tuple(),
+    make_relative: bool = False,
+) -> None:
+    """
+    Links all directories in src_path / dir_name to dst_path / dir_name.
+    """
+    for item in os.scandir(src_path):
+        if item.name not in ignore:
+            symlink_path = dst_path / item.name
+            if make_relative:
+                rel_or_abspath = os.path.relpath(Path(item.path), symlink_path.parent)
+            else:
+                rel_or_abspath = os.path.abspath(item.path)
+            symlink_path.symlink_to(rel_or_abspath)
```

Comment:
Why not simply use `iterdir`?
```suggestion
    for item in src_path.iterdir():
        if item.name not in ignore:
            symlink_path = dst_path / item.name
            if make_relative:
                rel_or_abspath = os.path.relpath(item, symlink_path.parent)
            else:
                rel_or_abspath = os.path.abspath(item)
            symlink_path.symlink_to(rel_or_abspath)
```